docs/Using-Fleet/REST-API.md

```diff
@@ -2538,7 +2538,94 @@ user by email.
 
 ---
 
-### Get host's mobile device management (MDM) and Munki information
+### Get host's mobile device management (MDM) information
```

Comment:
When changing a header's text, you need to update the table of contents somewhere above (the anchors are automatically generated from the text).

EDIT: I think this is actually a new endpoint, so it should be added in the `Hosts` TOC, same for the other new endpoint.